### PR TITLE
Improved the progress slider. Also the pre-exisiting logs will be cle…

### PR DESCRIPTION
**Pull request by Princu7**
…ared on removing or adding new zip file. Fixes the issue #957.
Behaviour of slider before changes
https://www.opentest.co/share/d25d2c70d4cb11e68531cf81a6426bba
After changes
https://www.opentest.co/share/63aa3ed0d4cb11e68531cf81a6426bba
@mariobehling @aayusharora Please review. Thanks!!

### CODE REVIEW

File: src/www/js/form.js [hunk 1]
```diff
@@ -140,17 +142,17 @@ $(document).ready(function () {
...
+    var divElem = $('<div></div>');
+    var paragraph = $('<p></p>');
     spanMess.css({'margin-left' : '5px'});
     aElem.css({'margin-left' : '5px'});
     aElem.attr({'data-toggle' : 'collapse', 'href' : '#error' + String(errorno)});
-    divElem.attr({'id' : 'error' + String(errorno)  , 'class' : "collapse"});
+    divElem.attr({'id' : 'error' + String(errorno)  , 'class' : 'collapse'});
     divElem.css({'color' : 'red'});
-    aElem.text("Know More");
+    aElem.text('Know More');
```
Comment (aayusharora) [inline src/www/js/form.js]: @Princu7 Can you please add some comments to the code lines.

File: src/backend/generator.js [hunk 2]
```diff
@@ -308,7 +308,7 @@ exports.createDistDir = function(req, socket, callback) {
     (done) => {
       logger.addLog('Info', 'Creating zip file of the event', socket);
       console.log("==================================Creating zip file");
-      if (emit) socket.emit('live.process', {donePercent:95, status: "Website is being generated"});
```
Comment (aayusharora) [inline src/backend/generator.js]: @Princu7 Why there a need of changing percent values?

File: src/www/js/form.js (hunk 1 above)
Comment (Princu7) [inline src/www/js/form.js]: Thanks. Will do it right away.

File: src/backend/generator.js (hunk 2 above)
Comment (Princu7) [inline src/backend/generator.js]: Intially, the percentage values were not in a sorted manner. 95 was coming before 90 as visible in the above code(90 was given for the mailing job which comes later). It was the reason why progress slider was not progressing smoothly.

Comment (aayusharora) [inline src/backend/generator.js]: Ok !! got this

File: src/www/js/form.js (hunk 1 above)
Comment (Princu7) [inline src/www/js/form.js]: @aayusharora I have added some comments. Please review them. Thanks!!

**Comment by aayusharora** [comment]
Thanks LGTM @Princu7 please Squash the commits.

**Comment by mariobehling** [comment]
As there have been some changes, conflicts appeared.

Please help to resolve this. Thanks!

**Comment by Princu7** [comment]
I made a mistake.

**Pull request closed without merging**